ui/src/app/pages/signed-in/nav-bar.tsx

```diff
@@ -119,7 +120,7 @@ export const NavBar = () => {
         accessTier={AccessTierShortNames.Controlled}
       />
       <StatusAlertBannerMaybe />
-      <TakeDemographicSurveyV2BannerMaybe />
+      {isUserFromUS(profile) && <TakeDemographicSurveyV2BannerMaybe />}
```

Comment:
since the component is already a "maybe", we can move the check inside it.  We can also move `shouldShowDemographicSurveyBanner()` to profile-utils and use that.

Reply:
this has been moved